### ISSUE TITLE
Tab buttons not keyboard accessible

### ISSUE THREAD

**Issue by tillprochaska** [HowTheyVote/howtheyvote#1023]
Tab buttons are currently not accessible via the tab key (only via arrow keys). Remove `tabindex="-1"`.
